backend/census_historical_migration/historic_data_loader.py

```diff
@@ -27,14 +28,15 @@ def load_historic_data_for_year(audit_year, page_size, pages):
         )
 
         for submission in page.object_list:
-            result = {"success": [], "errors": [], "transformations": []}
```

Comment:
We want to reset the result fields here otherwise, it will keep appending data to the previously run submission results. It could also be done in `end_to_end_core.py` at the beginning of run_end_to_end (which will make more sense)

Reply:
`migration_result.result = migration_result.DEFAULT_RESULT` below is the new reset, but I'll try moving it to run_end_to_end

Reply:
Moved the reset to e2ec